src/main/java/org/springframework/data/elasticsearch/core/DefaultIndexOperations.java

```diff
@@ -256,5 +259,29 @@ public boolean deleteTemplate(DeleteTemplateRequest deleteTemplateRequest) {
 				client -> client.indices().deleteTemplate(deleteIndexTemplateRequest, RequestOptions.DEFAULT).isAcknowledged());
 	}
 
+	@Override
+	public List<IndexInformation> getInformation() {
+		IndexCoordinates indexCoordinates = getIndexCoordinates();
+		GetIndexRequest request = requestFactory.getIndexRequest(indexCoordinates);
+
+		Map<String, Set<AliasData>> aliases = getAliasesForIndex(indexCoordinates.getIndexNames());
```

Comment:
this is executing a separate call to Elasticsearch for the aliases. But alias information is return in the GetIndex request, no need for a separate call here